apps/prairielearn/python/prairielearn.py

```diff
@@ -1812,3 +1813,8 @@ def index2key(i):
 
 def is_int_json_serializable(n: int) -> bool:
     return -((2**53) - 1) <= n <= 2**53 - 1
+
+
+def full_unidecode(input_str: str) -> str:
+    """Does unidecode of input"""
+    return unidecode(input_str.replace("\u2212", "-"))
```

Comment:
Should this be in prairielearn.py?

Reply:
Do you mean the python helper sympy? I guess it would be a little awkward to import it from there but prevents some duplicate logic. I'll move it there on my next push.

Reply:
I'm just considering that it makes sense to have this normalization be consistent across elements.

Reply:
Yes that makes sense, it's pretty easy to add this to the other elements once it's a helper function. I didn't do everything all at once just because I didn't have time to test every element.

Reply:
Switched to using uniform unidecode function here: https://github.com/PrairieLearn/PrairieLearn/pull/8634/commits/960769b0096267c004667233df31ee1718302445

Not sure if I'm in favor of switching every element over to this at once, one of those things that could cause weird minor issues I think. But easy enough to do one at a time now that this function is written.

Reply:
I just realized that this was indeed in prairielearn.py to begin with, sorry for the confusion, my first read I thought this was in an element... In any case, I agree with switching elements gradually.

My comment was just in the sense of ensuring that any element that *does* have normalization does it consistently. I don't care where the function lives, as long as all elements that do it call a centralized function for it. This would also allow us to more easily expand to other characters (e.g., [`\u2215`](https://www.compart.com/en/unicode/U+2215) is also a convertible character that is not supported by text-unidecode; while [`\u03c0`](https://www.compart.com/en/unicode/U+03C0) could potentially be converted to `pi`) in the future and have that change affect all elements that do the conversion.

Reply:
I definitely agree with that reasoning. The way this is currently set up in this PR should allow for this consistent normalization.